### PR TITLE
form player sentry dsn

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride @snopoke

### CODE REVIEW

File: ansible/roles/formplayer/vars/main.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 formplayer_build_dir: "{{ code_home }}/formplayer_build"
 formplayer_data_dir: "{{ encrypted_root }}/formplayer"
+# formplayer_sentry_dsn: ..defined in secret file
 # formplayer_db_name: ..defined in secret file
 hq_main_db_host: "{% for config in postgresql_dbs -%}
-  {%- if config.name == localsettings.PG_DATABASE_NAME -%}{{ config.host }}
+  {%- if config.name == localsettings.PG_DATABASE_NAME -%}
+  {%- if 'host' in config -%}{{ config.host }}
+  {% else %}{{ groups.postgresql.0 }}
+  {%- endif -%}
```
Comment (snopoke) [inline ansible/roles/formplayer/vars/main.yml]: Just learnt about Ansible's JSON Query filter (https://docs.ansible.com/ansible/playbooks_filters.html#json-query-filter):
```
hq_main_db_host: {{ postgresql_dbs|json_query('[*][?django_alias==\'default\'] | [0].host')|default(groups.postgresql.0) }}
```
Note: not tested - also fine to leave this as is - just thought I'd share

File: ansible/roles/formplayer/templates/application.properties.j2 [hunk 2]
```diff
@@ -8,6 +8,7 @@ server.port={{ formplayer_port }}
 user.suffix=commcarehq.org
 logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=INFO
 sqlite.dataDir={{ formplayer_data_dir }}/
+sentry.dsn={{ formplayer_sentry_dsn }}
```
Comment (snopoke) [inline ansible/roles/formplayer/templates/application.properties.j2]: what's the format of this? Could be useful to put in a comment.

File: ansible/roles/formplayer/vars/main.yml (hunk 1 above)
Comment (benrudolph) [inline ansible/roles/formplayer/vars/main.yml]: nice for now i'm just going to leave as is, but will use next time. very useful

File: ansible/roles/formplayer/templates/application.properties.j2 (hunk 2 above)
Comment (benrudolph) [inline ansible/roles/formplayer/templates/application.properties.j2]: https://github.com/dimagi/commcarehq-ansible/pull/718/commits/dbcd0cae2e50e1022757fd2fd40d40974828ab8a

**Comment by benrudolph** [comment]
fixed merge conflicts

**Pull request merged**